src/text-transform/transformers.ts

```diff
@@ -0,0 +1,94 @@
+import _ from 'lodash';
+
+import {ConstructorBlock} from '../models/constructor';
+import {fullTransform} from './utils';
+
+import {Block} from '../models';
+
+import {Lang} from '../utils/configure';
+
+import {Transformer} from './common';
+
+import {config, BlocksConfig} from './config';
+
+export type ContentTransformerProps = {
+    content: {
+        blocks?: ConstructorBlock[];
+        footnotes?: string[];
+    };
+    options: {
+        lang: Lang;
+        customConfig?: {};
+    };
+};
+
+function addRandomOrder(block: ConstructorBlock) {
+    if (block) {
+        if ('randomOrder' in block && block.randomOrder && 'children' in block && block.children) {
+            block.children = _.shuffle(block.children as ConstructorBlock[]);
```

Comment:
not to mutate incoming `block` param:
```
return {
  ...block,
  children: _.shuffle(block.children as ConstructorBlock[]);
}
```

Reply:
remove `addRandomOrder' because it's useless as a separate function, and insert this logic inside transformfunc